hyrax/Rakefile

```diff
@@ -5,4 +5,4 @@ require_relative 'config/application'
 
 Rails.application.load_tasks
 
-require 'solr_wrapper/rake_task' unless Rails.env.production?
+require 'solr_wrapper/rake_task' unless Rails.env.production? || ENV['IN_DOCKER'] == "true"
```

Comment:
no solr_wrapper in docker land. you don't want to launch solr twice on accident.